src/target/common/devo/msc2/usb_devo8.c

```diff
@@ -90,7 +93,7 @@ void USB_Enable(unsigned type, unsigned use_interrupt)
 
 void USB_Disable()
 {
-    gpio_set(GPIOB, GPIO10);
+    PowerOff();
```

Comment:
because original we didn't call PowerOff of USB stack, we didn't shutdown USB completely. we only did "USB_Cable_Config" in the following PowerOff function. that's why we endup increase by 4 bytes in ROM. 

  /* disable all interrupts and force USB reset */
  _SetCNTR(CNTR_FRES);
  /* clear interrupt status register */
  _SetISTR(0);
  /* Disable the Pull-Up*/
  USB_Cable_Config(DISABLE);
  /* switch-off device */
  _SetCNTR(CNTR_FRES + CNTR_PDWN);
  /* sw variables reset */
  /* ... */